web/sites/default/config/group.role.project-member.yml

```diff
@@ -14,48 +14,17 @@ permissions_ui: true
 permissions:
   - 'access group_media overview'
   - 'access group_node overview'
-  - 'create group_media:audio entity'
-  - 'create group_media:document entity'
-  - 'create group_media:image entity'
-  - 'create group_media:video entity'
   - 'create group_node:city_service entity'
```

Comment:
Looks like members for Project can edit, which is different from members in other groups.

Reply:
This is weird. City Services is not enabled for the project group type. I can't explain why this permission would be here, but it doesn't do anything. I guess I could manually remove it.

Reply:
Manual removal seems to have no side effects.